### PR TITLE
SJFM Commerce content changes

### LINKED ISSUE #415: SJFM Commerce content changes
Changes made to food offering product checkout flow before 2nd pilot begins
Check App Content Tasks section in document: https://docs.google.com/document/d/13QWweMko-8kVyvBzsCatyh44J6mdFScOekNqEcMKpuk/edit

### PR DESCRIPTION
**Pull request by DeeTheDev**
Fixes #415

### CODE REVIEW

File: webapp/src/pages/FoodList.js [hunk 1]
```diff
@@ -87,14 +87,7 @@ function Product({ product, offeringId }) {
   return (
     <Col xs={6} className="mb-4">
       <div className="position-relative">
-        <SumaImage
-          image={images[0]}
-          alt={name}
-          params={{ crop: "attention" }}
```
Comment (rgalanakis) [inline webapp/src/pages/FoodList.js:93]: Why did you change the crop behavior?

Comment (DeeTheDev) [reply webapp/src/pages/FoodList.js:93]: I had added that crop param but it didn't make sense so I left it at the default "center".

File: lib/suma/tasks/bootstrap.rb [hunk 2]
```diff
@@ -212,51 +212,62 @@ def setup_products
 
   def setup_market_offering_product
     market_name = "St. Johns Farmers Market"
+    market_address = Suma::Address.lookup(
+      address1: "N Charleston Avenue &, N Central Street",
```
Comment (rgalanakis) [inline lib/suma/tasks/bootstrap.rb:216]: `N Charleston Ave & N Central St`

File: lib/suma/tasks/bootstrap.rb [hunk 3]
```diff
@@ -212,51 +212,62 @@ def setup_products
...
     if offering.fulfillment_options.empty?
       offering.add_fulfillment_option(
         type: "pickup",
         ordinal: 0,
         description: Suma::TranslatedText.create(
-          en: "Redeem this voucher at #{market_name} (July 16 2023).
-               For more information check the product details.",
-          es: "Reclame este boleto en #{market_name} (16 julio 2023). Para
-               más información verifique los detalles del producto.",
+          en: "Redeem your vouchers at #{market_name} July 15, 2023",
```
Comment (rgalanakis) [inline lib/suma/tasks/bootstrap.rb:237]: `vouchers on`

File: webapp/public/locale/en/strings.json [hunk 4]
```diff
@@ -170,7 +170,7 @@
     "total": "{{total, sumaCurrency}}",
     "unclaimed_order_history_title": "Claim Orders",
     "unclaimed_order_history_intro": "Orders that are ready to claim will appear below. When you arrive to the order pickup location, please allow a staff member to help you \"slide to claim\" your order.",
-    "unclaimed_orders": "Unclaimed orders ({{unclaimedOrdersCount}})"
+    "unclaimed_orders": "Claimed orders ({{unclaimedOrdersCount}})"
```
Comment (rgalanakis) [inline webapp/public/locale/en/strings.json:173]: `Claim orders` (also change the key to reflect the verb usage)

File: webapp/src/pages/FoodDetails.js [hunk 5]
```diff
@@ -65,7 +65,6 @@ export default function FoodDetails() {
         image={product.images[0]}
         alt={product.name}
         className="w-100"
-        params={{ crop: "attention" }}
```
Comment (rgalanakis) [inline webapp/src/pages/FoodDetails.js:68]: Restore these.

File: webapp/src/modules/externalLinks.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ const externalLinks = {
   fundingSourcesLink: "https://mysuma.org/sumaplatform#funding-sources",
```
Comment (rgalanakis) [inline webapp/src/modules/externalLinks.js:3]: Delete this if unused.
